lib/src/base-styles/_nx-lists.scss

```diff
@@ -133,104 +130,64 @@
   }
 
   .nx-list__item {
-    @include semi-bold();
-
-    color: $nx-text-color-light;
-
     &.selected {
-      background-color: $nx-grey-3;
-      color: $nx-text-color;
+      background-color: #d3e6ff;
       pointer-events: none;
     }
 
-    &:not(.nx-list__item--empty):not(.nx-disabled):not([disabled]) {
+    &:not(.nx-list__item--empty):not(.disabled):not([disabled]),
+    &:not(.nx-list__item--error) {
       &:hover, &:focus {
-        background-color: $nx-color-blue-96;
-        border-top: $nx-border-row-hover;
-        border-bottom: $nx-border-row-hover;
-        color: $nx-text-color;
+        background-color: #f1f3fa;
         cursor: pointer;
         outline-style: none; // removes :focus outline
 
         .fa-chevron-right {
-          color: $nx-color-blue-41;
+          color: #045dd1;
         }
       }
 
       &:active {
-        background-color: $nx-grey-1;
-        border-top: $nx-border-row-hover;
-        border-bottom: $nx-border-row-hover;
+        background-color: #d3e6ff;
       }
     }
 
     &.nx-list__item--empty {
       @include regular();
 
-      border-top: $nx-border; // assume it's always the first item
       color: $nx-text-color-light;
     }
 
-    &.nx-error {
-      color: $nx-text-color-error;
-
-      &:not(.nx-disabled):not([disabled]) {
-        border: $nx-border-error;
-
-        &:hover, &:focus, &:active {
-          background-color: $nx-fail-color-bg;
-          border: $nx-border-error;
-          color: $nx-text-color-error;
-          cursor: default;
-        }
-      }
-    }
-
     .nx-list__subtext {
       pointer-events: none; // Fix for IE11 :active state display
     }
 
-    &.nx-list__item--disabled {
+    &.disabled {
       color: $nx-text-color-disabled;
-      cursor: default;
 
       &:hover {
-        border-top: 1px solid transparent;
-        border-bottom: $nx-border-row;
+        background-color: transparent;
+        cursor: default;
       }
     }
   }
 
-  .fa-chevron-right {
-    margin-right: 3px;
-    margin-top: 1px;
+  .nx-icon {
+    align-self: center;
+    grid-column: 2;
+    grid-row: 1;
```

Comment:
Zeplin shows the chevron being vertically centered in taller rows.  I think the grid placement here needs to span both rows

Reply:
> Zeplin shows the chevron being vertically centered in taller rows. I think the grid placement here needs to span both grid rows

Tweaked here: https://github.com/sonatype/sonatype-react-shared-components/pull/141/commits/c5aa6499a051116f2101b2d7be86452d4062f373